app/src/renderer/main.js

```diff
@@ -30,12 +30,26 @@ window.addEventListener(`unhandledrejection`, function(event) {
 window.addEventListener(`error`, function(event) {
   Raven.captureException(event.reason)
 })
-Vue.config.errorHandler = error => {
+
+Vue.config.errorHandler = (error, vm, info) => {
+  console.error(`An error has occurred: ${error}
+
+Guru Meditation #${info}`)
```

Comment:
wut

Reply:
`Guru Meditation #` is what Amiga computers displayed when they crashed.  I'm just showing off how old I am.  😉